sdk/python/feast/data_source.py

```diff
@@ -354,11 +355,12 @@ def get_table_column_names_and_types(
 
     def __init__(
         self,
-        name: str,
-        event_timestamp_column: str,
-        bootstrap_servers: str,
-        message_format: StreamFormat,
-        topic: str,
+        *args,
+        name: Optional[str] = None,
+        event_timestamp_column: Optional[str] = "",
```

Comment:
any reason why the default for this is `""` and `None` for the others?

Reply:
I believe, previous iterations on BigQuerySource, SnowflakeSource etc. used "" for the timestamp columns. I just followed that convention. Do you think I should change it to None?

Reply:
I don't think it matters too much, as long as it's consistent - if it's consistent with other implementations that's fine